добавлено первое задание

gorbunov-d opened the pull request.

gorbunov-d committed:
добавлено первое задание

AlexKitIt commented on sb_unit_tests/tests/test_bun.py:
```diff
@@ -0,0 +1,18 @@
+        ("sesame bun", 199.99),
+    ],
+)
+def test_bun_getters(name, price):
```
Нужно исправить: здесь и далее по тестам: тестовые методы в модулях объединяются в тестовые классы, необходимо создать в каждом тестовом модуле тестовые классы и разместить в них методы, даже если тестовый метод только один

AlexKitIt commented on sb_unit_tests/tests/test_bun.py:
```diff
@@ -0,0 +1,18 @@
+def test_bun_getters(name, price):
+    bun = Bun(name, price)
+    assert bun.get_name() == name
+    assert bun.get_price() == price
```
Нужно исправить: здесь и далее: это юнит тесты, на каждый метод должен быть отдельный тест

AlexKitIt commented on sb_unit_tests/tests/test_burger.py:
```diff
@@ -0,0 +1,75 @@
+)
+
+
+@pytest.fixture
```
Нужно исправить: фикстуры живут в модуле conftest

gorbunov-d committed:
Перенес фикстуру `burger` в `tests/conftest.py`.
- Объединил тесты в классы: `TestBun`, `TestIngredient`, `TestBurger`, `TestDatabase`.
- Разделил проверки геттеров на отдельные тесты; добавил параметризацию.
- Использовал моки в `TestBurger` для расчета цены.
- Сделал тесты независимыми и убрал дубли.
- Отчет покрытия настроен через pytest-cov в `pytest.ini` (xml + html); покрытие 100% (>70%).
- Подчистил лишнее: импорты и зависимости; `requirements.txt` минимален.
